mw-webapp/src/component/button/Button.cy.tsx

```diff
@@ -0,0 +1,36 @@
+import {Button} from "src/component/button/Button";
+
+const BUTTON_VALUE = "Click button";
+
+describe("<Button />", () => {
+  it("renders", () => {
+    const STUB_FUNCTION = cy.stub();
+    cy.mount(
+      <Button
+        value={BUTTON_VALUE}
+        onClick={STUB_FUNCTION}
+      />);
+  });
+
+  it("uses custom text for the button", () => {
```

Comment:
What does mean custom? Do you have default? Describe what do you test like : something should happened in case if something... Etc.

Reply:
Rewrote